packages/react-scripts/template/src/app/app.test.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './App';
+import App from './app';
```

Comment:
I reverted back to lowercase because clients with linux / windows might have problems working together